### DISCUSSION TITLE
Node tooltip font

### DISCUSSION THREAD

**Discussion by quickfold** [freeplane/freeplane#2151] [Help] [answered]
The node tooltip font is different than my node and menu font. I looked for a place to set the node tooltip font and couldn't find it. I expected the node tooltip to be a style like details or attributes. The current node tooltip font is hard for me to read. Is there a way to change it?

![image](https://github.com/user-attachments/assets/e1707ec9-6d61-4b6e-9866-55c2e2b10287)

**Comment by euu2021** [comment]
related request: https://github.com/freeplane/freeplane/issues/1728

**Comment by dpolivaev** [comment]
I checked the mind map you attached to the related request, and there Freeplane behaves as expected: it uses the same font family and font size as nodes at zoom 100%. This logic comes from assumption that 100% is generally the most comfortable for you, and having a different zoom is only needed in special cases and should not be applied to tooltips.

**Reply by quickfold** [reply]
The node and tooltip may be the same font on your computer, but please see the image in my original post. It is clearly not the same font on my computer. e.g. Note the "L" in the word "Details" is a straight vertical line while the "L" in the word "Default" has serifs on the top and bottom. How can I help find out why the font is not the same on my computer but it is on yours?

**Reply by kenfhill84083** [reply]
I agree.  The tooltip font in your screenshot is obviously a serif font while the nodes use san serif.  I, too, struggle with serif fonts on a screen.
Here, however, the tooltip font and node font are both san serif.  My eyeballs are not sufficiently calibrated to tell if they are the same font.

![freeplane _ display node styles in tool tips](https://github.com/user-attachments/assets/bc2a213c-3f2d-4fc4-921f-42b40ec6d9f5)

**Reply by quickfold** [reply]
@kenfhill84083 the tooltip and node font are indeed different. Look at the "e" in both. The horizontal bar in the "e" is higher in the node text than in the tooltip text.

**Comment by dpolivaev** [comment]
I can’t interpret the first screenshot posted by @quickfold without the corresponding mind map and a screenshot of the resulting tooltip. To analyze the issue, I need both the mind map and the screenshots to fully understand what you’re seeing. Missing any part makes it impossible for me to address the ticket, which is very frustrating.

**Reply by quickfold** [reply]
Here is a map that is created from the standard1.6.mm template, and I just added one node and hovered over it to show the tooltip. Zoom is at 100%. You can see the node text is a sans-serif and the tooltip text is a serif font. Please let me know any other information you need for troubleshooting.
[tooltip.zip](https://github.com/user-attachments/files/18271701/tooltip.zip)


![image](https://github.com/user-attachments/assets/ecf79cbc-cd37-455b-87fe-c4d8003d2e69)

**Comment by dpolivaev** [comment, marked as answer]
I found multiple bugs related specifically to displaying node styles in the node tooltip.
Localizing the problem was possible only after I got your mind map and screenshot.
Preview 1.12.9_01 is fixing all found bugs.
